test_util/launch.py

```diff
@@ -61,40 +68,103 @@ def delete(self, info):
     def ssh_from_config(self, info):
         raise NotImplementedError()
 
-    def test(self, info):
-        raise NotImplementedError()
+    def test(self, info, test_cmd):
```

Comment:
This is an abstract class because there will be an AzureLauncher, AwsVpcLauncher, OnpremLaunhcer, and probably a GceVpcLauncher which must all use this same API. Creating a provider generic abstract API was one of the driving design decisions for all of dcos-launch